### ISSUE TITLE
Why is buyer-legal-type a resource it is not used in F06? 

### ISSUE THREAD

**Issue by muricna** [OP-TED/ted-rdf-mapping#287]
Resource sheet: should indicate either the forms the authority tables are used in or the ones they are exempt form.
This may also need to be reflected in the technical mappings

**Comment by csnyulas** [comment]
Indeed, the Resources sheet of the Conceptual Mapping (CM) lists all the resources that are necessary for the technical mappings to be able to be executed. Since all form-specific technical mappings are generated from the same source, but each with an individual set of section-specific mapping module, the subject maps for all resources will be present, but for forms that do not use certain resources there will be no predicate-object maps, and therefore they will never be used (i.e. fired).

In fact in our Master CM, from which all form-specific CM are generated, we noted that all resources are necessary (for technical reasons) in all forms, but we also highlighted, with red font, which are the resources that will not be practically necessary to generate the result for a certain form.
![image](https://user-images.githubusercontent.com/2441046/213373220-b02ad6c3-94f3-4513-a97c-3f992fd144d8.png)

We wanted to transfer this information, through the highlighting(s), to the form-specific conceptual maps. We did it for some (e.g. F20 or F23), but for many (including F06), we have not (or it got lost from during the various version upgrades).

Would it be a satisfactory solution to have the **unused** (but technically necessary) resources highlighted in red? This is how would look like for the different forms

![image](https://user-images.githubusercontent.com/2441046/213374552-816eeb93-ba6f-41d6-858a-ec1b06d34ada.png)

![image](https://user-images.githubusercontent.com/2441046/213374616-4e71d18d-14e9-47af-bcc3-c2dc8a1f1275.png)

![image](https://user-images.githubusercontent.com/2441046/213374681-47ae4886-585c-47ec-94d4-51cb2a284d91.png)

![image](https://user-images.githubusercontent.com/2441046/213374748-6a6c4ada-fa2f-45ef-83a0-6919710a786c.png)

![image](https://user-images.githubusercontent.com/2441046/213374798-0cf17d96-1ffc-4f88-8d04-b4966af5de2a.png)

![image](https://user-images.githubusercontent.com/2441046/213374854-b848c36e-f1b9-46d9-9bf6-1590ca41ebc6.png)

![image](https://user-images.githubusercontent.com/2441046/213374884-4ace51ff-ddd7-4acc-9590-90eba3d3aa20.png)


**Note:** This is already implemented starting with v6.0.0 of the conceptual mapping!
